### DISCUSSION TITLE
estimate uncertainty?

### DISCUSSION THREAD

**Discussion by thangckt** [MDIL-SNU/SevenNet#108] [General]
Dear all, 
Do you have any idea to efficiently estimate uncertainty of the model?

Thanks.

**Comment by YutackPark** [comment]
There is a interesting pull request regarding uncertainty estimation in MACE
https://github.com/ACEsuit/mace/pull/601

The method seems very cost-effective, and I'm looking on it :)

**Reply by thangckt** [reply]
@YutackPark, great :)

Hope to see its availability in 7net too.

**Comment by thangckt** [comment]
hi @YutackPark 

Do you have any update on this ?

Thanks.

**Reply by YutackPark** [reply]
Hi @thangckt , no update about this.
